src/components/Header/web3/SelectWallet.tsx

```diff
@@ -163,17 +163,16 @@ function Web3StatusInner() {
                 text={
                   <Text style={{ fontSize: '12px', textAlign: 'left', whiteSpace: 'normal' }}>
                     <Trans>
-                      You are not signed in with this wallet address. If you wish, you can
-                      <Text
-                        as="span"
+                      You are not signed in with this wallet address. If you wish, you can{' '}
+                      <span
```

Comment:
any reason for this?

Reply:
dont know why, maybe some default style of <Text as='span'>

before
<img width="226" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/cafcc3ec-c3b7-4c96-8e2e-463ce1be6fd5">
==========
after
<img width="245" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/922ec192-774d-4413-82a4-bf315a0858f0">